Azure/communication-ui-library-ios
# [Feature] Call Eviction

## Purpose
Added call eviction and its UI.

https://user-images.githubusercontent.com/90668345/164570075-f9ad5a0c-2823-4840-b808-70434b06346a.mp4

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```




* Test the code
* Launch app, join a group call / teams call
* On another desktop, join a group call / teams call, and remove the remote participant from the call
* Go back to app, see if the user is removed from the call (setup screen shown with error message)
* Follow **What to Check** section

## What to Check
Verify that the following are valid
* Check if the user is removed from the call in the app and app navigates user back to setup screen
* Check if the error message shows correctly in the setup screen, displaying a banner with call eviction message.

## Other Information
